roles/database/tasks/set-environment-variables.yml

```diff
@@ -1,21 +1,15 @@
 ---
-- name: Test if a PostgreSQL password file is present.
-  stat:
-    path: "{{ psql_env.PGPASSFILE }}"
-  register: password_file
-  changed_when: false
-  
-- name: Create the PostgreSQL password file for a local database.
+- name: Create the PostgreSQL password file for database connection.
   template:
     src: pgpass_conf.j2
-    dest: "{{ psql_env.PGPASSFILE }}"
+    dest: "{{ fpsd_database_psql_env.PGPASSFILE }}"
     mode: "0600"
-  when: not password_file.stat.exists
+    force: no # Don't clobber preexisting file, since it may contain auth secrets
```

Comment:
More specifically, it will always clobber secrets you don't want clobbered if you reprovision without setting `fpsd_database_password` and `/tmp/passwordfile` is no longer there.